interface/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "spl-token-interface"
+version = "1.0.0"
+description = "Solana Program Library Token Interface"
+documentation = "https://docs.rs/spl-token-interface"
+readme = "README.md"
+authors = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+arrayref = "0.3.9"
+bytemuck = { version = "1.23.1", features = ["derive"] }
+num-derive = "0.4"
+num_enum = "0.7.4"
+num-traits = { workspace = true }
```

Comment:
nit: I know this was here before, but since all of the other deps here don't use the workspace version, may as well stay consistent and define the version here explicitly too